external/include/mbedtls/config.h

```diff
@@ -3026,6 +3026,11 @@
 #include MBEDTLS_USER_CONFIG_FILE
 #endif
 
+/* override some previous default for STDK. */
+#if defined(CONFIG_STDK_IOT_CORE)
+#include "../../stdk/tizenrt-wrapper/mbedtls_config.h"
```

Comment:
Is file path also the final result:  put it in STDK, not in external/include/medtls ?

Reply:
as considered that include/mbedtls have so many mbedtls related hpp files, it's not clear to add an external module customized config file there. 
for alternative, I moved it to external/include/stdk, and this path would also be refined.